source/builtin_macros/src/syntax.rs

```diff
@@ -164,6 +175,40 @@ impl Visitor {
                     typed.attrs.push(mk_verus_attr(token.span, quote! { proof }));
                 }
             }
+
+            // Check for Ghost(x) or Tracked(x) argument
+            use syn_verus::PatType;
+            if let FnArgKind::Typed(PatType { pat: box pat, .. }) = &mut arg.kind {
+                let mut tracked_wrapper = false;
+                let mut wrapped_pat_id = None;
+                if let Pat::TupleStruct(tup) = &pat {
+                    let ghost_wrapper = path_is_ident(&tup.path, "Ghost");
+                    tracked_wrapper = path_is_ident(&tup.path, "Tracked");
```

Comment:
I imagine we are considering these (here and elsewhere) to be keywords, rather than constructor names in patterns, right? By that I mean: if someone hides `Ghost` or `Tracked` with a definition/import, we still treat these as Ghost/Tracked "patterns".

Reply:
Yes, they are treated as keywords (or at least "soft keywords") rather than resolved names.